src/lib.rs

```diff
@@ -782,17 +782,15 @@ impl<T> EventListener<T> {
     /// use event_listener::{Event, EventListener};
     ///
     /// let event = Event::new();
-    /// let listener = EventListener::new(&event);
```

Comment:
This seems to be a breaking change.

Reply:
Whoops, I accidentally pushed from a branch where I was experimenting with https://github.com/smol-rs/event-listener/issues/91.